src/grafana_service.ts

```diff
@@ -21,6 +22,82 @@ export class DatasourceUnavailable extends DataKitError {};
 
 const CHUNK_SIZE = 50000;
 
+export class TimeSeriesPoint {
+  constructor(
+    public columns: string[],
+    public values: number[]
+  ){}
+}
+
+export class TimeSeriesChunk {
```

Comment:
It looks very suspicious that you have two very similar classes; TimeSeriesPoint  and TimeSeriesPoint